src/Clock.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component } from 'react';
+
+type State = {
+  today: string;
+};
+
+type Props = {
+  clockName: string,
+};
+
+const formatedDate = (): string => {
```

Comment:
function should be named as a function, starting with a verb that describes what the function does
```suggestion
const getFormattedDate = (): string => {
```